scripts/deploy.sh

```diff
@@ -90,9 +91,14 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
       echo "osd_directory: true" | tee -a $RPCD_VARS
       echo "osd_directories:" | tee -a $RPCD_VARS
       echo "  - /var/lib/ceph/osd/mydir1" | tee -a $RPCD_VARS
-      sed -i "s/glance_default_store:.*/glance_default_store: rbd/" /etc/openstack_deploy/user_variables.yml
-      echo "nova_libvirt_images_rbd_pool: vms" | tee -a /etc/openstack_deploy/user_variables.yml
-      echo "cinder_ceph_client_uuid:"  | tee -a /etc/openstack_deploy/user_secrets.yml
+      echo "glance_default_store: rbd" | tee -a /etc/openstack_deploy/user_osa_variables_defaults.yml
```

Comment:
I disagree, we have too many files as it is. I can just about except the aio and gating files because the allow it was the easiest way to keep the existing variable precedence without having the merge variables together but here it makes no sense to me. What is the point in creating an additional file location. This is for an AIO. These values are used to override the defaults. The contract with support is not to modify the overrides files on customer deployments. The overrides file is exactly the place these changes should go. If I'm working with an AIO I should have a set up that is as close to a production deployment as possible.